modern/src/resources/l10n/en.json

```diff
@@ -99,6 +99,8 @@
     "sharedImport": "Import",
     "sharedColumns": "Columns",
     "sharedDropzoneText": "Drag and drop a file here or click",
+    "sharedReload": "Reload",
+    "updateAvailable": "New update available, click on the reload button to update.",
```

Comment:
This should have a prefix. Maybe it should all be under "settings" prefix?

Reply:
Also I think the wording can be simplified. If there's a clear reload button, there's no need to tell that you have to reload.

Reply:
Please include a screenshot.

Reply:
Simplified the wording, see attached screenshot of a mobile view.
![Screenshot from 2023-11-06 09-47-12](https://github.com/traccar/traccar-web/assets/858821/4642fe3c-54b1-4b94-9d6e-bb5fa62c0214)

Reply:
There is also a possibility of outlined button
![Screenshot from 2023-11-06 13-47-12](https://github.com/traccar/traccar-web/assets/858821/6d105aec-9987-4f6f-b0be-cc674284f9e5)